duh

wafehling opened the pull request:
<!-- Impstation note: there's no need to read all the official contributing guidelines,
but please DON'T combine upstream changes and your own changes.
Make separate pull requests for separate changes. -->
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->

## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

## Media
<!--
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.
-->

## Requirements
<!--
Due to influx of PR's we require to ensure that PR's are following the correct guidelines.

Please take a moment to read these if its your first time.

Check the boxes below to confirm that you have in fact seen these (put an X in the brackets, like [X]):
-->
- [ ] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->

**Changelog**
<!-- Impstation note: we have our own changelog now, so please DO use this section! -->
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up. Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

hivehum committed:
made the sec blade sec contraband

hivehum committed:
repoman hardsuit

hivehum committed:
Merge pull request #1481 from EpicToastTM/map-updates-01252025

Fix missing wire on Barratry

Commit:
tiny fix. shocked this hasn't crashed the server

hivehum committed:
Merge pull request #1486 from ruddygreat/tiny-mail-fix

Tiny mail fix. shocked this hasn't crashed the server.

mqole committed:
Merge pull request #1483 from hivehum/markings-edits

back end marking edits

mqole committed:
displacement fixes

hivehum committed:
rewriting history

mqole committed:
Merge pull request #1488 from hivehum/one-line

history is being rewritten

DinnerCalzone committed:
cog fixes

mqole committed:
Merge pull request #1484 from hivehum/space-blade-edits

space blade tweaks

mqole committed:
Merge pull request #1485 from hivehum/cybersun-repoman-suit

repoman hardsuit

ImpstationBot committed:
Automatic changelog update

mqole committed:
oopsy

hivehum committed:
Merge pull request #1487 from mqole/displace

displacement fixes

ImpstationBot committed:
Automatic changelog update

DinnerCalzone committed:
more fixes

hivehum committed:
Merge pull request #1489 from mqole/oop

one line change

mqole committed:
Merge pull request #1490 from DinnerCalzone/stations

Cog fixes

DinnerCalzone committed:
solid public airlock sprite tweak

DinnerCalzone committed:
Merge branch 'glass-airlock' of https://github.com/DinnerCalzone/imp-station-14 into glass-airlock

dayn-e committed:
Rework BarkAccentSystem to sound more scooby-like; remove conversion of the letter L to R

12rabbits committed:
Update BorgSystem.MMI.cs

hivehum committed:
Merge pull request #1437 from 12rabbits/unborgable_reagents

Unborgable brain reagents

ImpstationBot committed:
Automatic changelog update

kipdotnet committed:
the monkey

hivehum committed:
Merge pull request #1495 from kipdotnet/bees--2!

monkeys/kobolds can pilot shuttles

ImpstationBot committed:
Automatic changelog update

Sleiphros committed:
general prettification + fix the sauna with space heaters + non-liquid air cans

DinnerCalzone committed:
fixed missing pa control board

hivehum committed:
Merge pull request #1496 from DinnerCalzone/stations

Fix missing PA computer board on Marathon

ImpstationBot committed:
Automatic changelog update

Sleiphros committed:
Merge branch 'impstation:master' into testing

kipdotnet committed:
the monkey

EpicToastTM committed:
Initial commit

EpicToastTM committed:
Fix YAML issues

kipdotnet committed:
logistics access

kipdotnet committed:
& who could forget mail boy

kipdotnet committed:
& the

EpicToastTM committed:
Add Ripley to Union

EpicToastTM committed:
forgot the clamp

EpicToastTM committed:
Add mechs to the hand labeler whitelist

hivehum committed:
Merge pull request #1498 from kipdotnet/ther-monkey-2

New salvage shuttle for purchasing + bargain bin buff

ImpstationBot committed:
Automatic changelog update

hivehum committed:
Merge pull request #1497 from Sleiphros/testing

Minor derelict tweaks and fixes

ImpstationBot committed:
Automatic changelog update

hivehum committed:
Merge pull request #1501 from EpicToastTM/map-updates

Add Ripley to Union Station

Kandiyaki committed:
re-rotate heretic (also rebalance secret weights to compensate)

dayn-e committed:
Add imp comments

dayn-e committed:
Merge branch 'impstation:master' into improved-dog-accent

irismessage committed:
Fix role ping on discord webhook

irismessage commented:
what's going on in here

irismessage commented:
I get an email whenever a GitHub actions run fails in this fork, could you disable those please

iaada committed:
Merge remote-tracking branch 'imp-downstream/master' into feature/partial/LensSlot

# Conflicts:
#	Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml
#	Resources/Prototypes/Entities/Clothing/Eyes/hud.yml
#	Resources/Prototypes/Entities/Clothing/Eyes/specific.yml
#	Resources/Prototypes/Entities/Structures/Machines/lathe.yml
#	Resources/Prototypes/Traits/disabilities.yml

iaada committed:
migrate, remove dupes

mqole committed:
Merge pull request #1503 from Kandiyaki/impStation

re-rotate heretic

mqole committed:
Merge pull request #1504 from irismessage/webhook-allowed-mentions

Fix role ping on discord webhook

ImpstationBot committed:
Automatic changelog update

mqole committed:
Merge pull request #1502 from EpicToastTM/ripley-label

Add mechs to the hand labeler whitelist

ImpstationBot committed:
Automatic changelog update

iaada committed:
yml update.

DinnerCalzone committed:
attribution fix

iaada committed:
un-fuck BlurryVisionSystem

iaada committed:
these comments shouldn't be in this pr

iaada committed:
comment

iaada committed:
reorginize

formlessnameless committed:
Merge branch 'master' of https://github.com/impstation/imp-station-14 into dev

iaada committed:
helper function

iaada committed:
commit

DinnerCalzone committed:
amber power change

DinnerCalzone committed:
fix missing tile

EpicToastTM committed:
Remove jaws of life, add second thruster

honeyed-lemons committed:
update

honeyed-lemons committed:
Oopsies forgot the sink :3

kipdotnet committed:
yuuuup

kipdotnet committed:
whitespace

EpicToastTM committed:
Remove thruster from evac pod

kipdotnet committed:
why is there localization files in the textures directory

AvianMaiden committed:
law desc + examine

hivehum committed:
Merge pull request #1506 from iaada/imp/comments

Comments in lathe.yml

AvianMaiden committed:
numbered entries

hivehum committed:
Merge pull request #1510 from AvianMaiden/Machine-state-your-laws

Law Board Descriptions!

ImpstationBot committed:
Automatic changelog update

DinnerCalzone committed:
remove non-working code

iaada committed:
more stuff idk

iaada committed:
more make the code not as bad

DinnerCalzone committed:
anotha one

kipdotnet committed:
little

hivehum committed:
Merge pull request #1514 from kipdotnet/gonk

Tck'tck multitool + salv access

ImpstationBot committed:
Automatic changelog update

DinnerCalzone committed:
supermatter fix

DinnerCalzone committed:
chapelroid disposals, dirt fix

DinnerCalzone committed:
supermatter tweak

DinnerCalzone committed:
rename pump

DinnerCalzone committed:
change gas pump

DinnerCalzone committed:
oh god I'm so used to mapping mode i didnt realize how weird a freezer connected into a pipe under a floor tile looks im so sorry

kipdotnet committed:
fuck ypou

Darkmajia committed:
Merge pull request #1508 from DinnerCalzone/stations

Fix Bagel supermatter

honeyed-lemons committed:
wah

hivehum committed:
Merge pull request #1511 from DinnerCalzone/tank-slots

Remove non-functional code from U.A.P.

formlessnameless committed:
Merge branch 'master' of https://github.com/impstation/imp-station-14 into dev

honeyed-lemons committed:
Update gas-pipe-sensor.ftl

formlessnameless committed:
Merge branch 'stable' of https://github.com/space-wizards/space-station-14 into dev

hivehum committed:
Merge pull request #1519 from honeyed-lemons/gas-sensors-fix

New Gas Pipe sensor prototypes and bug fix

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
i fix

formlessnameless committed:
fuck 10 years we imp

hivehum committed:
Merge pull request #1518 from kipdotnet/kill-it

Comment out the flagship wreck

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
yaml stop yell

formlessnameless committed:
Merge branch 'master' of https://github.com/impstation/imp-station-14 into dev

mqole committed:
Merge pull request #1494 from dayn-e/improved-dog-accent

Dog Accent Rework

ImpstationBot committed:
Automatic changelog update

mqole committed:
gonna give courier lowpop salv access, while we're here

mqole committed:
Merge pull request #1505 from iaada/imp/LensSlot

LensSlot update

mqole committed:
Merge pull request #1500 from kipdotnet/mailtime

Logistics access group

mqole committed:
Merge pull request #1507 from DinnerCalzone/amber-station

Amber engineering update

mqole committed:
Merge pull request #1509 from honeyed-lemons/synth-ship-updates

Synthesis Ship updates

ImpstationBot committed:
Automatic changelog update

mqole committed:
Merge pull request #1515 from DinnerCalzone/xeno

Minor Marathon update

mqole committed:
Merge pull request #1517 from DinnerCalzone/janitor

Minor Cog update

formlessnameless committed:
Merge branch 'master' of https://github.com/impstation/imp-station-14 into dev

mqole committed:
Merge pull request #1465 from iaada/imp/relentless-penguin

Relentless grenade penguin

ImpstationBot committed:
Automatic changelog update

EpicToastTM committed:
Move new disaster evac pod shuttle (with two emergency EVAs instead of one) to _Impstation

formlessnameless committed:
fixes but linter still hates me

mqole committed:
Merge pull request #1499 from EpicToastTM/visitor-pdas

Give visitors proper accesses (and other relevant tweaks)

ImpstationBot committed:
Automatic changelog update

mqole committed:
minor ii rules note

Pinkbat5 committed:
Ive  been in the life hack business doing some hard thinking & i found out u can drink water from an old empty handle of gin fot a while and when every body knows its just water u can start putting gin in i5 again so nobody goes crazy at u anymore

DinnerCalzone committed:
my repo is haunted

DinnerCalzone committed:
fix missing lv cable

DinnerCalzone committed:
add brigmedic

hivehum committed:
Merge pull request #1521 from mqole/hell

minor ii rules note

ImpstationBot committed:
Automatic changelog update

DinnerCalzone committed:
it never ends this shit

formlessnameless committed:
this should do it

DinnerCalzone committed:
what do you mean none of the external airlocks were linked to bolt each other

hivehum committed:
Merge pull request #1520 from formlessnameless/dev

Merge Stable Branch From Upstream

DinnerCalzone committed:
salvage updates

DinnerCalzone committed:
incorporate upstream amber changes

DinnerCalzone committed:
oops forgot solar station beacons

formlessnameless committed:
missed a line so every locker had an enemy in it

formlessnameless committed:
Merge pull request #1527 from formlessnameless/dev

Maint Lockers Always Containing Enemies Bugfix

ImpstationBot committed:
Automatic changelog update

TheGrimbeeper committed:
DNA scramble artifact effect no longer update records computer

TheGrimbeeper committed:
fixed error

hivehum committed:
Merge pull request #1491 from DinnerCalzone/glass-airlock

Public airlock sprite tweak

hivehum committed:
Merge pull request #1524 from DinnerCalzone/xeno

Origin station update

hivehum committed:
Merge pull request #1523 from DinnerCalzone/tank-slots

Oasis Update

ImpstationBot committed:
Automatic changelog update

hivehum committed:
mind role info for every ghost role that didnt have it already

hivehum committed:
and the two goob antags

hivehum committed:
hop needed courier access lol

hivehum committed:
wardrobe restock works on courierdrobe

hivehum committed:
Merge pull request #1526 from DinnerCalzone/barrel

Update Amber to incorporate latest upstream version

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
Merge pull request #1530 from hivehum/actually-ghost-role-touchup

added mindRoles/roleTypes to everything that didnt have one

formlessnameless committed:
Merge pull request #1532 from hivehum/hop-courier

hop needed courier access lol

formlessnameless committed:
Merge pull request #1533 from hivehum/courierdrobe-restock

wardrobe restock works on courierdrobe

formlessnameless committed:
Merge pull request #1528 from TheGrimbeeper/artifacts

DNA scramble artifact effect no longer update records computer

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
Merge pull request #1522 from Pinkbat5/hosjetpacksize

Security Jetpack Buff

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
sec jetpack changes

formlessnameless committed:
fuck it we ball

formlessnameless committed:
Merge pull request #1534 from formlessnameless/dev

Small Changes to Sec Jetpacks

TheGrimbeeper committed:
Removed redundant high damage trigger, radioactive chems now trigger radiation node

TheGrimbeeper committed:
attribution

TheGrimbeeper committed:
Thank you online linter, local one hates giving details

TheGrimbeeper committed:
Merge remote-tracking branch 'upstream/master' into artifacts

TheGrimbeeper committed:
fixed clownspiders throwing an error every frame they move

this does not give them a 'moving' sprite, it just prevents it from throwing an error because it inherits from MobSpiderAngryBase which expects a movement sprite. The sprite given is the same when moving and not moving

formlessnameless committed:
Merge pull request #1537 from TheGrimbeeper/clownspider-error-every-frame

fixed clownspiders throwing an error every frame they move

ImpstationBot committed:
Automatic changelog update

widgetbeck committed:
rummaging componentization (dupe of upstream)

widgetbeck committed:
goblin rummaging

honeyed-lemons committed:
Grey Resprite

Commit:
added the buyer species condition

honeyed-lemons committed:
update full

hivehum committed:
sparklechicken

kipdotnet committed:
tcktck gun

hivehum committed:
removed dupe thief beacon

widgetbeck committed:
correct copyright

kipdotnet committed:
sound

honeyed-lemons committed:
Update textures and ear markings to match outlines

honeyed-lemons committed:
Mail update

formlessnameless committed:
Merge pull request #1543 from hivehum/thief-beacon-dupe

removed dupe thief beacon

formlessnameless committed:
Merge pull request #1544 from honeyed-lemons/update-mail

Mail Buffs!!!!!!!! Everyone its mail buff time

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
Merge pull request #1540 from honeyed-lemons/species-sprites

Gray Sprite Touchups

formlessnameless committed:
Merge pull request #1539 from ruddygreat/traitor-item-species-conditional

Add a species buyer condition for stores

ImpstationBot committed:
Automatic changelog update

hivehum committed:
spiter get witer

formlessnameless committed:
Merge pull request #1538 from widgetbeck/goblins

Goblin Rummaging

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
Merge pull request #1535 from TheGrimbeeper/artifacts

Radioactive chems for radiation artifact triggers

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
Merge pull request #1541 from hivehum/sparklechicken

Vox Overhaul/Markings Update

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
Merge pull request #1545 from hivehum/spider-whitewashing

arachnid resprite

ImpstationBot committed:
Automatic changelog update

EpicToastTM committed:
Update Meta to the upstream version

formlessnameless committed:
give aghosts an eye slot

formlessnameless committed:
Merge branch 'master' of https://github.com/impstation/imp-station-14 into dev

formlessnameless committed:
Merge pull request #1547 from formlessnameless/dev

Give Admin Ghosts An Eye Slot

formlessnameless committed:
Merge pull request #1546 from EpicToastTM/map-updates

Update Meta to the upstream version

honeyed-lemons committed:
Forks gate

EpicToastTM committed:
Update Union Station

DinnerCalzone committed:
fix shuttle eta timers

Kandiyaki committed:
67% more heretic per heretic

hivehum committed:
Merge pull request #1551 from EpicToastTM/map-updates

(Union Station) Minor fixes

hivehum committed:
Merge pull request #1552 from DinnerCalzone/evac

Fix Oasis, Plasma, Elkridge evac timers

hivehum committed:
Merge pull request #1542 from kipdotnet/gonk

tcktck gun update

ImpstationBot committed:
Automatic changelog update

DinnerCalzone committed:
shuffling stuff around

DinnerCalzone committed:
stuff

formlessnameless committed:
lings enter stasis on death automatically and updated stasis fail messages

formlessnameless committed:
Merge branch 'master' of https://github.com/impstation/imp-station-14 into dev

formlessnameless committed:
Merge pull request #1553 from Kandiyaki/impStation

increase amount of heretics

formlessnameless committed:
Merge pull request #1555 from DinnerCalzone/bagel

Minor Bagel Update

formlessnameless committed:
Merge pull request #1556 from DinnerCalzone/cog

Minor Cog update

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
fix boat

formlessnameless committed:
Merge pull request #1558 from formlessnameless/master

Fix Boat

ImpstationBot committed:
Automatic changelog update

honeyed-lemons committed:
fix silly error

DinnerCalzone committed:
Merge pull request #1550 from honeyed-lemons/fork-gate

Forks gate

formlessnameless committed:
king soda and jungle juice changes

formlessnameless committed:
Merge branch 'impstation:master' into master

formlessnameless committed:
Merge pull request #1559 from formlessnameless/master

Jungle Juice Prevention Act

ImpstationBot committed:
Automatic changelog update

Darkmajia committed:
gray accent regex and uap fix

Darkmajia committed:
is she stupid

hivehum committed:
minor vox fixes

hivehum committed:
sorry ghoul lol

Darkmajia committed:
dino slight rebalance, hardsuit, commander

formlessnameless committed:
Merge pull request #1561 from Darkmajia/gray-accent-stuff

gray accent regex + uap fix

ImpstationBot committed:
Automatic changelog update

formlessnameless committed:
Merge pull request #1564 from hivehum/christmas-is-over

removed present anom

formlessnameless committed:
Merge pull request #1563 from hivehum/vox-fix

minor vox fixes

ImpstationBot committed:
Automatic changelog update

Darkmajia committed:
more dino wars

TheGrimbeeper committed:
UAP from artifact foam

TheGrimbeeper committed:
Merge remote-tracking branch 'upstream/master' into artifacts

formlessnameless committed:
Merge pull request #1566 from TheGrimbeeper/artifacts

UAP foam from artifacts

formlessnameless committed:
Merge pull request #1565 from Darkmajia/dino-wars-final-remix

DINO WARS: FINAL REMIX

ImpstationBot committed:
Automatic changelog update

12rabbits committed:
Reinstated buckle checks in StandingStateSystem.cs

Darkmajia committed:
Merge pull request #1569 from 12rabbits/unbuckle_crit_fix

Fix unbuckling after crit

ImpstationBot committed:
Automatic changelog update

Darkmajia committed:
Merge pull request #1557 from formlessnameless/dev

Changelings Enter Stasis Automatically On Death

ImpstationBot committed:
Automatic changelog update
